wg-naming/recommendations/template.md

```diff
@@ -0,0 +1,17 @@
+# Recommendation: Old Term -> New Term
+
+**Status:** Accepted|Draft|Other
+
+## Recommendation 
+
+- Formalize recommendation in title 
+- Provide a brief, 1-3 sentence summary of reasoning behind this change
+- Provide alternate recommendation(s) to the one in the title if needed 
+
+## Context/Precedents 
+
+If existing, provide any research, links to PR(s) from other communities, etc. that provide precedent for this decision. 
+
+## Consequences
+
+Starting with a [Hound](https://cs.k8s.io/) keyword search, note any major changes that will result by making this change.
```

Comment:
Unsure if this is the right PR to start this discussion on, but #5211 mentions owner and stakeholder groups as well - I feel like that should also fall under either "consequences" or perhaps a separate heading for who is affected by this change. That way there's more information for the eventual ADR.

[I don't actually know the process for creating ADRs (if someone could point me - that would be amazing!) but currently 5211 makes it sound like someone will propose a recommendation using this template and WG Naming will create an ADR for it based on that after review. Since the ADR requires review from owner/stakeholder groups as well (and those aren't included in this template yet) it seems the onus would then be on WG Naming to seek out/research which groups would be affected.]

Reply:
Hmm. I think this is a good point. I'm going to add it to the agenda for the next meeting?